templates/zerver/emails/email_base.html

```diff
@@ -1,17 +1,350 @@
+            }
+            table[class=body] p,
+            table[class=body] ul,
+            table[class=body] ol,
+            table[class=body] td,
+            table[class=body] span,
+            table[class=body] a {
+                font-size: 16px !important;
+            }
+
+            table[class=body] .wrapper {
+                padding: 75px 10px 10px 10px;
+            }
+
+            table[class=body] .article {
+                padding: 10px !important;
+            }
+            table[class=body] .content {
+                padding: 0 !important;
+            }
+            table[class=body] .container {
+                padding: 0 !important;
+                width: 100% !important;
+            }
+            table[class=body] .main {
+                border-left-width: 0 !important;
+                border-radius: 0 !important;
+                border-right-width: 0 !important;
+            }
+            table[class=body] .btn table {
+                width: 100% !important;
+            }
+            table[class=body] .btn a {
+                width: 100% !important;
+            }
+            table[class=body] .img-responsive {
+                height: auto !important;
+                max-width: 100% !important;
+                width: auto !important;
+            }
+        }
+        /* -------------------------------------
+        PRESERVE THESE STYLES IN THE HEAD
+        ------------------------------------- */
+
+        @media all {
+            .ExternalClass {
+                width: 100%;
+            }
+            .ExternalClass,
+            .ExternalClass p,
+            .ExternalClass span,
+            .ExternalClass font,
+            .ExternalClass td,
+            .ExternalClass div {
+                line-height: 100%;
+            }
+            .apple-link a {
+                color: inherit !important;
+                font-family: inherit !important;
+                font-size: inherit !important;
+                font-weight: inherit !important;
+                line-height: inherit !important;
+                text-decoration: none !important;
+            }
+        }
+        </style>
     </head>
-    <body>
-        <table width="80%" style="align:center; max-width:800px" align="center">
+    <body class="">
+        <table border="0" cellpadding="0" cellspacing="0" class="body">
             <tr>
-                <td style="font-size:16px; font-family:Helvetica;">
-                    {% block content %}{% endblock %}
+                <td>&nbsp;</td>
+                <td class="container">
+                    <div class="content">
+                        <div class="graphic-wrapper">
+                            {% block illustration %}{% endblock %}
+                        </div>
+                        <!-- START CENTERED WHITE CONTAINER -->
+                        <span class="preheader">This is preheader text. Some clients will show this text as a preview.</span>
```

Comment:
I'm not sure whether my PR about preheaders will merge first and make this moot, but this should probably be a jinja2 template `{% block preheader %}{% endblock %}` so we don't default to sending emails with placeholder preheader text.